### PR TITLE
Remove tests that rely on g6 services

### PR DESCRIPTION
**Pull request by idavidmcdonald**
So we can have g6 expired on preview.

Note, the majority of these tests are covered by the new master branch smoke tests.

These tests identified from the following test report when g6 was expired.
https://ci.beta.digitalmarketplace.service.gov.uk/job/functional-tests-preview-legacy/77/functional_test_report/

### CODE REVIEW

File: features/admin/admin_journey.feature [hunk 1]
```diff
@@ -22,20 +22,6 @@ Scenario: As an admin user who has logged in to Digital Marketplace, I wish to s
   And I click the search button for 'service_id'
   Then I am presented with the summary page for that service
 
-Scenario: As an admin user who has logged in to Digital Marketplace, I wish to search for services by DUNS number
```
Comment (TheDoubleK) [inline features/admin/admin_journey.feature]: Feels like this wouldn't be so hard to patch up with a G7/G8 supplier?

File: features/admin/admin_journey.feature [hunk 2]
```diff
@@ -22,20 +22,6 @@ Scenario: As an admin user who has logged in to Digital Marketplace, I wish to s
...
-  Given I have logged in to Digital Marketplace as a 'Administrator' user
-  When I enter '930123456789' in the 'supplier_duns_number' field
-  And I click the search button for 'supplier_duns_number'
-  And I am taken to the 'Suppliers' page
-  And I click the 'Services' link
-  Then  I am presented with the 'Services' page for the supplier 'DM Functional Test Supplier'
-  And I can see all listings ordered by lot name followed by listing name
```
Comment (TheDoubleK) [inline features/admin/admin_journey.feature]: If we're removing these steps then we should also delete the unused step definitions. (At least, I assume things like  "I can see all listings ordered by lot name followed by listing name" and "I am presented with the service page for that specific listing" are specific to just these steps.)

File: scripts/test_dependencies.sh [hunk 3]
```diff
@@ -29,7 +29,7 @@ echo -e "\033[0;32mAll services are up\033[0m"
 
 test_framework_status "g-cloud-4" "expired"
 test_framework_status "g-cloud-5" "expired"
-test_framework_status "g-cloud-6" "live"
+test_framework_status "g-cloud-6" "expired"
 test_framework_status "g-cloud-7" "live"
 test_framework_status "g-cloud-8" "live"
 test_framework_status "digital-outcomes-and-specialists" "live"
```
Comment (TheDoubleK) [inline scripts/test_dependencies.sh]: Do we care about the state of DOS2?

File: features/admin/admin_journey.feature (hunk 1 above)
Comment (idavidmcdonald) [inline features/admin/admin_journey.feature]: This one should be covered by the master tests - https://github.com/alphagov/digitalmarketplace-functional-tests/blob/f5372b4e008e148319ad4366521077a882a90d2c/features/admin/search.feature#L14

**Comment by pcraig3** [comment]
Also, which tests aren't being replaced?  Is the difference between the ones we still cover and the ones we don't based on user types (we have buyer+supplier+admin users in here) or by particular actions (like removing services)?

File: scripts/test_dependencies.sh (hunk 3 above)
Comment (idavidmcdonald) [inline scripts/test_dependencies.sh]: I don't think we have any dependancies on it at the moment... although they may soon come

**Comment by idavidmcdonald** [comment]
Coverage lost roughly appears to be:
- Admin searching by DUNS number and supplier name prefix (we used to have tests for all admin types (admin, category and sourcing). In our new tests we only check with one of these admin types.

- Admin/users changing services from public/private and removing them.

- User searching for services by service ID, service name, service description, keywords field

- Some scenarios regarding navigating to service pages

- Some scenarios about supplier dashboard and viewing their services

File: features/admin/admin_journey.feature (hunk 2 above)
Comment (idavidmcdonald) [inline features/admin/admin_journey.feature]: Thanks, you are right. New commit should deal with removing all unused steps.

**Comment by TheDoubleK** [comment]
Have these run and passed green locally at all? If so it is traditional to add :sparkles: to your pull request.

**Comment by idavidmcdonald** [comment]
Note, these tests are passing when I run them on preview from my own laptop 
image

**Comment by TheDoubleK** [comment]
OK let's do it
:burn::burn::burn::burn:
:+1:
:burn::burn::burn::burn:

**Pull request merged**